app/src/main/kotlin/com/simplemobiletools/gallery/activities/ViewPagerActivity.kt

```diff
@@ -510,34 +519,13 @@ class ViewPagerActivity : SimpleActivity(), ViewPager.OnPageChangeListener, View
         bmp.compress(file.getCompressionFormat(), 90, out)
     }
 
-    private fun copyFile(sourcePath: String, destinationPath: String) {
-        var inputStream: FileInputStream? = null
-        var outputStream: FileOutputStream? = null
-        try {
-            inputStream = FileInputStream(sourcePath)
-            outputStream = FileOutputStream(destinationPath)
-            inputStream.copyTo(outputStream)
-            scanPath(destinationPath) {}
-        } catch (ignored: FileNotFoundException) {
-        } finally {
-            inputStream?.close()
-            outputStream?.close()
-        }
-    }
-
-    private fun rotateFileByExif(path: String) {
-        val exif = ExifInterface(path)
-        val orientation = exif.getAttributeInt(ExifInterface.TAG_ORIENTATION, ExifInterface.ORIENTATION_NORMAL)
-        val orientationDegrees = (degreesForRotation(orientation) + mRotationDegrees) % 360
+    private fun saveRotation(input: File, output: File) {
+        copyFile(input, output)
+        val exif = ExifInterface(output.absolutePath)
+        var orientation = exif.getAttributeInt(ExifInterface.TAG_ORIENTATION, ExifInterface.ORIENTATION_NORMAL)
+        var orientationDegrees = (degreesForRotation(orientation) + mRotationDegrees) % 360
```

Comment:
you can make these 2 variables "val" instead of "var", it is showing a hint at android studio

Reply:
Aaah yeah, true. Used to be vars, didn't change.

I'm still used to Eclipse, and all the Android Studio hints and helps are still really annoying for me. Thanks for showing me the ropes!